examples/zustand-flowchart/package-lock.json

```diff
@@ -265,6 +266,34 @@
         "react-dom": ">=17"
       }
     },
+    "node_modules/@reactflow/background/node_modules/zustand": {
+      "version": "4.5.5",
```

Comment:
Seems like reactflow uses Zustand v4, so now both versions will get bundled. Is there a way to either upgrade reactflow to a later version that also uses Zustand, or to keep using Zustand v4 in this example? (Hopefully the upgrade to V5 is backward compatible?)

Reply:
Yes it's backward compatible. I tested locally and everything was looking fine. I will check if I can updated `reactflow` just in case 👍🏻

Reply:
So I checked and `reactflow` do not supports yet `zustand` v5.  But as stated it's fully backward compliant. 
I just updated `reactflow` to its latest version to remove a deprecation warning.

So it's fine from my side 🙂